app/routes/_layout+/settings.template.index.tsx

```diff
@@ -0,0 +1,293 @@
+import type { Template } from "@prisma/client";
+import { TierId } from "@prisma/client";
+import type {
+  MetaFunction,
+  LoaderFunctionArgs,
+  ActionFunctionArgs,
+} from "@remix-run/node";
+import { json, redirect } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import { ErrorBoundryComponent } from "~/components/errors";
+import { EmptyState } from "~/components/list/empty-state";
+import { ListHeader } from "~/components/list/list-header";
+import { ListItem } from "~/components/list/list-item";
+import { Badge } from "~/components/shared";
+import { ControlledActionButton } from "~/components/shared/controlled-action-button";
+import { Table, Td, Th } from "~/components/table";
+import { TemplateActionsDropdown } from "~/components/templates/template-actions-dropdown";
+import { db } from "~/database";
+import { commitAuthSession, requireAuthSession } from "~/modules/auth";
+import { requireOrganisationId } from "~/modules/organization/context.server";
+import { makeActive, makeDefault, makeInactive } from "~/modules/template";
+import { assertIsPost } from "~/utils";
+import { appendToMetaTitle } from "~/utils/append-to-meta-title";
+import { sendNotification } from "~/utils/emitter/send-notification.server";
+import { ShelfStackError } from "~/utils/error";
+import { canCreateMoreTemplates } from "~/utils/subscription";
+
+export const loader = async ({ request }: LoaderFunctionArgs) => {
+  const authSession = await requireAuthSession(request);
+  const { organizationId } = await requireOrganisationId(authSession, request);
+  const { userId } = authSession;
+
+  const user = await db.user.findUnique({
```

Comment:
I am not sure why we are doing this here. You can get userId from session and then query templates by themself. Should be faster.

Reply:
Resolved